ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -421,6 +421,88 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-serial,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/e2e-aws-single-node
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-machine-config-operator-master-e2e-aws-single-node
+    rerun_command: /test e2e-aws-single-node
```

Comment:
can you add optional: false

Reply:
Actually wanted to add `optional: true` and `always_run: false` but forgot. Will fix that.

This is because for now those tests don't pass because of some OpenShift components not being ready for single node and misbehaving.